charts/gitops-runtime/templates/_components/gitops-operator/_default_values.tpl

```diff
@@ -79,14 +79,22 @@ securityContext:
   capabilities:
     drop:
       - "ALL"
-command: []
+command:
+  - /manager
```

Comment:
I'm not sure that command needs to be set at all - the image has ENTRYPOINT set to /manager so shouldn't it use that by default? 🤔